clients/admin-ui/cypress/e2e/systems-plus.cy.ts

```diff
@@ -168,15 +168,26 @@ describe("System management with Plus features", () => {
       cy.getByTestId("input-name").should("not.be.disabled");
     });
 
-    it("allows changes to data uses for non-GVL vendors", () => {
+    it("don't allow editing declaration name after creation", () => {
       cy.getSelectValueContainer("input-name").type("L{enter}");
       cy.getByTestId("save-btn").click();
       cy.wait(["@postSystem", "@getSystem", "@getSystems"]);
       cy.getByTestId("tab-Data uses").click();
       cy.getByTestId("add-btn");
       cy.getByTestId("delete-btn");
       cy.getByTestId("row-functional.service.improve").click();
-      cy.getByTestId("input-name").should("not.be.disabled");
+      cy.getByTestId("input-name").should("be.disabled");
+    });
+
+    it("don't allow editing data uses after creation", () => {
+      cy.getSelectValueContainer("input-name").type("L{enter}");
+      cy.getByTestId("save-btn").click();
+      cy.wait(["@postSystem", "@getSystem", "@getSystems"]);
+      cy.getByTestId("tab-Data uses").click();
+      cy.getByTestId("add-btn");
+      cy.getByTestId("delete-btn");
```

Comment:
The gets for `add-btn` and `delete-btn` are a little redundant here; those are only in the previous test to make sure that we show the interface for changing when the form isn't locked, which isn't really relevant here.